### ISSUE TITLE
Gnome 42 compatibility

### ISSUE THREAD

**Issue by smhg** [trifonovkv/ReminderAlarmClock#20]
@trifonovkv thank you for the recent Gnome 40 release.
Would it be possible to also 'allow' Gnome 42 (default on Ubuntu 22.04 LTS)? I've tested current `master` on this version and it seems to work flawlessly.

**Comment by nE0sIghT** [comment]
As I remember Shell 42 should not check compatibility version.
Is it really issue?

**Comment by smhg** [comment]
I didn't know about that. Searching on https://extensions.gnome.org/ does not show it unless I enable 'All versions' in search.

**Comment by smhg** [comment]
Also, I installed it manually by downloading the zip (master) from GitHub. I did change metadata.json so it contains `"shell-version": ["40"],`.
It works the first time and when opening the Extensions app, it shows an update is/was installed and will be enabled after reboot. After that update, it doesn't work anymore:
![image](https://user-images.githubusercontent.com/2411348/173128346-5aca2c33-2f90-4df3-9e54-a4d6f641db31.png)

Does this make any sense?
